sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -476,7 +503,14 @@
    "metadata": {},
    "source": [
     "### Explaining Predictions\n",
-    "There are expanding business needs and legislative regulations that require explainations of _why_ a model mades the decision it did. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+    "There are expanding business needs and legislative regulations that require explanations of _why_ a model made the decision it did. SageMaker Clarify uses SHAP to explain the contribution that each input feature makes to the final decision."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Kernel SHAP algorithm requires baseline dataset (also known as background dataset). Baseline dataset type shall be the same as dataset type, and baseline samples shall only include features. By definition, `baseline` should either be a S3 URI to the baseline dataset file, or an in-place list of samples. In this case we chose the latter, and only put a single sample to the list."
```

Comment:
* requires *a* baseline
* By type do you mean schema? 
* Can you explain what this single sample is?

Reply:
Thanks.

> * require a baseline

Fixed in the new revision.

> By type do you mean schema

In the new revision I updated the sentence to "Baseline dataset type shall be the same as `dataset_type` of `DataConfig`"

> Can you explain what this single sample is?

In the new revision I updated the sentence to "In this case we chose the latter, and put the first sample of the test dataset the list."